### PR TITLE
Add changelog for 0.3.0

### PR DESCRIPTION
**Pull request by gnufied**
Add changelog for 0.3.0

cc @msau42

### CODE REVIEW

File: CHANGELOG-0.3.0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# v0.3.0
```
Comment (msau42) [inline CHANGELOG-0.3.0.md]: Can you actually make the filename based on minor versions, not patch verisons? So CHANGELOG-0.3.md

Comment (gnufied) [reply CHANGELOG-0.3.0.md]: Do we want rename existing changelog files too?

Comment (msau42) [reply CHANGELOG-0.3.0.md]: yeah that sounds good

**Pull request merged**